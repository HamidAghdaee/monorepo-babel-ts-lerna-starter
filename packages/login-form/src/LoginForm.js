import * as React from "react";
const wrapText = (text) => React.createElement("span", null,
    "$",
    text);
export const LoginForm = () => React.createElement("div", null,
    "$",
    wrapText("Login form"));
