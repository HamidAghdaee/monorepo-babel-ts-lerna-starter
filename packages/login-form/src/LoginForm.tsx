import * as React from "react";
import styled from "styled-components";

const BoldText = styled.span`
  font-weight: bold;
`;
const wrapText = (text: string) => <BoldText>${text}</BoldText>;

export const LoginForm: React.SFC = () => <div>${wrapText("Login form")}</div>;
