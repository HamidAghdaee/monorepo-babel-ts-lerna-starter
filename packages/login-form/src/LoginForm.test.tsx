import { mount } from 'enzyme';
import * as React from 'react';

import { LoginForm } from './LoginForm';

describe('LoginForm', () => {
  test('should match snapshot and styles', () => {
    expect(mount(<LoginForm />)).toMatchSnapshot();
  });
});
