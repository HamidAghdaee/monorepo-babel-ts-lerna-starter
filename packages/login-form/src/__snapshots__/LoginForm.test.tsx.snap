// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LoginForm should match snapshot and styles 1`] = `
.c0 {
  font-weight: bold;
}

<Component>
  <div>
    <styled.span>
      <span
        className="c0"
      >
        Login form
      </span>
    </styled.span>
  </div>
</Component>
`;
